### ISSUE TITLE
Cannot Create User On Filament Admin

### ISSUE THREAD

**Issue by itsbreadd** [chiiya/filament-access-control#36]
When creating users through the filament form in the admin panel, you cannot create a new user unless you assign them no role.
You have to save the user without, then edit the role after save.

```
SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'model_id' cannot be null (Connection: mysql, SQL: insert into `model_has_roles` (`model_id`, `model_type`, `role_id`) values (?, Chiiya\FilamentAccessControl\Models\FilamentUser, 2))
```

**Comment by DoniyorSamandarov** [comment]
@itsbreadd you found solve this to issue

**Comment by reaboivictor** [comment]
Hi, I was struggling with this too, I think it is related to [filament](https://github.com/filamentphp/filament) because I got the error with `filament/filament v2.17.32` but after downgrading it to `filament/filament v2.17.21` (this is what we are using on another project where everything works) the error was gone. 
I don't know which version introduces this error. But maybe it would be a good idea to open an issue to [filament](https://github.com/filamentphp/filament)

**Comment by chiiya** [comment]
Hey, sorry it took me some time to look at this.

Should be fixed in 1.7.1
